OpenEdXMobile/res/drawable/auto_complete_end_icon_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@drawable/ic_expand_less" android:state_checked="true" />
+    <item android:drawable="@drawable/ic_expand_more" />
```

Comment:
We already have `ic_drop_down` and `ic_drop_up` in our codebase. Can't we utilize that?